docs/planning/frontend/src/pages/EvaluatePage.jsx

```
@@ -0,0 +1,250 @@
+          <p className="text-sm text-gray-500 mt-1">
+            {selectedFragrance
+              ? `Selected: ${selectedFragrance.name} by ${selectedFragrance.brand}`
+              : "Type to search existing fragrances, or enter a new one (we'll look it up)"
+            }
+          </p>
+        </div>
+
+        {/* Brand (optional if not selected from search) */}
+        {!selectedFragrance && (
+          <div>
+            <label className="label">Brand (optional)</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)